fe2-android/app/src/androidTest/java/com/github/dedis/popstellar/ui/detail/event/rollcall/RollCallEventCreationFragmentTest.java

```diff
@@ -1,116 +1,231 @@
-  //        .perform(PickerActions.setDate(YEAR, MONTH_OF_YEAR, DAY_OF_MONTH));
-  //    onView(withId(android.R.id.button1)).perform(click());
-  //    onView(withId(R.id.start_date_edit_text)).check(matches(withText(DATE)));
-  //
-  //    onView(withId(R.id.start_time_edit_text)).perform(click());
-  //    onView(withClassName(Matchers.equalTo(TimePicker.class.getName())))
-  //        .perform(PickerActions.setTime(HOURS, MINUTES));
-  //    onView(withId(android.R.id.button1)).perform(click());
-  //    onView(withId(R.id.start_time_edit_text)).check(matches(withText(TIME)));
-  //
-  //    onView(withId(R.id.roll_call_confirm)).check(matches(isEnabled()));
-  //  }
-  //
-  //  @Test
-  //  @Ignore("TODO: solve issue with GithubActions Emulator")
-  //  public void confirmAddsEventToEventList() {
-  //    final String RANDOM_EVENT_TITLE = "Random roll call title";
-  //    onView(withId(R.id.roll_call_title_text)).perform(typeText(RANDOM_EVENT_TITLE));
-  //
-  //    onView(withId(R.id.start_date_edit_text)).perform(click());
-  //    onView(withClassName(Matchers.equalTo(DatePicker.class.getName())))
-  //        .perform(PickerActions.setDate(YEAR, MONTH_OF_YEAR, DAY_OF_MONTH));
-  //    onView(withId(android.R.id.button1)).perform(click());
-  //    onView(withId(R.id.start_date_edit_text)).check(matches(withText(DATE)));
-  //
-  //    onView(withId(R.id.start_time_edit_text)).perform(click());
-  //    onView(withClassName(Matchers.equalTo(TimePicker.class.getName())))
-  //        .perform(PickerActions.setTime(HOURS, MINUTES));
-  //    onView(withId(android.R.id.button1)).perform(click());
-  //    onView(withId(R.id.start_time_edit_text)).check(matches(withText(TIME)));
-  //
-  //    onView(withId(R.id.roll_call_confirm)).check(matches(isEnabled()));
-  //    onView(withId(R.id.roll_call_confirm)).perform(click());
-  //
-  //    activityScenarioRule
-  //        .getScenario()
-  //        .onActivity(
-  //            activity -> {
-  //              PoPApplication app = (PoPApplication) activity.getApplication();
-  //              List<Event> events =
-  //                  app.getCurrentLao().map(Lao::getEvents).orElse(new ArrayList<>());
-  //              List<String> eventsName =
-  //                  events.stream().map(Event::getName).collect(Collectors.toList());
-  //              Assert.assertThat(RANDOM_EVENT_TITLE, isIn(eventsName));
-  //            });
-  //  }
+
+  private static final int YEAR;
+  private static final int MONTH_OF_YEAR;
+  private static final int DAY_OF_MONTH;
+  private static final String DATE;
+
+  private static final int HOURS = 12;
+  private static final int MINUTES = 15;
+  private static final String TIME = "" + HOURS + ":" + MINUTES;
+
+  static {
+    // Make sure the date is always in the future
+    Calendar today = Calendar.getInstance();
+    today.add(Calendar.MONTH, 13);
+
+    YEAR = today.get(Calendar.YEAR);
+    MONTH_OF_YEAR = today.get(Calendar.MONTH);
```

Comment:
Calendar.MONTH start at 0
```suggestion
    MONTH_OF_YEAR = today.get(Calendar.MONTH) + 1;
```

Reply:
done